pkg/syncer/admissionhandler/cnscsi_admissionhandler.go

```diff
@@ -85,10 +86,14 @@ func (h *CSISupervisorWebhook) Handle(ctx context.Context, req admission.Request
 	log.Debugf("CNS-CSI validation webhook handler called with request: %+v", req)
 
 	resp = admission.Allowed("")
-	if containerOrchestratorUtility.IsFSSEnabled(ctx, common.TKGsHA) {
-		if req.Kind.Kind == "PersistentVolumeClaim" {
+	if req.Kind.Kind == "PersistentVolumeClaim" {
+		if containerOrchestratorUtility.IsFSSEnabled(ctx, common.TKGsHA) {
 			resp = validatePVCAnnotation(ctx, req)
+			if !resp.Allowed {
+				return
+			}
 		}
+		resp = validatePVCVolumeHealthAnnotation(ctx, req)
```

Comment:
Is there a FSS for Volume Health? We check if it is enabled if the Volume health feature is not GA'ed

Reply:
+1 on this

Reply:
Added the FSS check.